include/scrimmage/simcontrol/SimControl.h

```diff
@@ -276,6 +462,33 @@ class SimControl {
 
     DelayedTask reseed_task_;
     bool limited_verbosity_;
+
+ private:
+    bool take_step();
```

Comment:
Moved many functions that were public to private to reduce complexity of SimControl API for external users.

Reply:
good idea. Thanks.